src/CardPL.java

```diff
@@ -0,0 +1,85 @@
+import java.util.Arrays;
+import java.util.List;
+public class CardPL {
+
+    private Suit suit;
+    private FiguresPL figures;
+
+    private List<FiguresPL> createList() {
+//próbowałam z petli forache ale program nie kompilował, wiec mam cos takiego, da się to zrobic latwiej?
```

Comment:
Jesli wartosc zaimplementujesz jako enuma to  bedziesz mogla uzyc metody `values()` zeby moc przeiterowac  sie po wszystkich wartosci uzywajac pętli:

```
        for (Suit suit : suits) {
            for (Figure figure : figures) {

            }
        }
```